dissect/target/plugins/scrape/qfind.py

```diff
@@ -1,21 +1,32 @@
 from __future__ import annotations
 
 import codecs
+import hashlib
 import re
 import string
-import sys
 from pathlib import Path
 from typing import TYPE_CHECKING, Callable
 
-from dissect.cstruct import utils
-
-from dissect.target.helpers.scrape import recover_string
+from dissect.target.helpers.record import TargetRecordDescriptor
 from dissect.target.plugin import Plugin, arg, export
 
 if TYPE_CHECKING:
-    from dissect.target.container import Container
-    from dissect.target.target import Target
-    from dissect.target.volume import Volume
+    from collections.abc import Iterator
+
+
+re_NOFLAG = 0  # re.NOFLAG is Python 3.11 and newer only
+
+QFindMatchRecord = TargetRecordDescriptor(
+    "qfind/match",
+    [
+        ("string", "disk"),
+        ("varint", "offset"),
+        ("string", "needle"),
+        ("string", "codec"),
+        ("string", "match"),
```

Comment:
```suggestion
        ("bytes", "match"),
```
I think this should be bytes too?